convert2rhel/checks.py

```diff
@@ -422,10 +422,14 @@ def _bad_kernel_version(kernel_release):
 
 def _bad_kernel_package_signature(kernel_release):
     """Return True if the booted kernel is not signed by the original OS vendor, i.e. it's a custom kernel."""
-    kernel_pkg = run_subprocess(
+    kernel_pkg, return_code = run_subprocess(
         ["rpm", "-qf", "--qf", "%{NAME}", "/boot/vmlinuz-%s" % kernel_release], print_output=False
-    )[0]
+    )
     logger.debug("Booted kernel package name: {0}".format(kernel_pkg))
+    # add a case for when the kernel package isn't returning any package info
+    if return_code is 0:
+        logger.critical("The running kernel is not from a rpm")
```

Comment:
Should we test that this branch was actually used? I don't think that the current unit tests are handling this

Reply:
The user won't know what to do to fix this. The message they should see is here: https://github.com/oamg/convert2rhel/blob/v0.25/convert2rhel/checks.py#L380. That means you need to change this from critical to warning and return True from the function right away so that the linked message gets displayed.

Also I suggest making the message more meaningful to the user, similar to this message https://github.com/oamg/convert2rhel/blob/v0.25/convert2rhel/checks.py#L434.

```suggestion
    vmlinuz_path = "/boot/vmlinuz-%s" % kernel_release
    ...
    if return_code is 0:
        logger.warning("The running kernel %s is not owned by any installed package. It needs to be owned by a package signed by %s." % {vmlinuz_path,os_vendor})
        return True
```

Reply:
thank you guys, I will work on these changes!